<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormSettings</class>
 <widget class="QDialog" name="FormSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="m_buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Apply|QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QListWidget" name="m_listSettings">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="movement">
      <enum>QListView::Movement::Static</enum>
     </property>
     <property name="selectionRectVisible">
      <bool>false</bool>
     </property>
     <property name="currentRow">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QStackedWidget" name="m_stackedSettings">
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="BaseLineEdit" name="m_txtFindInSettings">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="placeholderText">
      <string>Search in settings</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BaseLineEdit</class>
   <extends>QLineEdit</extends>
   <header>baselineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_listSettings</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
